src/routes/console/project-[project]/messaging/+layout.ts

```diff
@@ -2,9 +2,34 @@ import Breadcrumbs from './breadcrumbs.svelte';
 import Header from './header.svelte';
 import type { LayoutLoad } from './$types';
 
-export const load: LayoutLoad = async () => {
+import { queries, queryParamToMap } from '$lib/components/filters';
+import { PAGE_LIMIT } from '$lib/constants';
+import { getLimit, getPage, getQuery, getSearch, pageToOffset } from '$lib/helpers/load';
+import { sdk } from '$lib/stores/sdk';
+import { Query } from '@appwrite.io/console';
+
+import type { Load } from '@sveltejs/kit';
+
+// I **think** this should be enough, tbh
+const loadProviders = async ({ url, route }: Parameters<Load>[0]) => {
+    const page = getPage(url);
+    const limit = getLimit(url, route, PAGE_LIMIT);
+    const offset = pageToOffset(page, limit);
+    const query = getQuery(url);
+    const parsedQueries = queryParamToMap(query || '[]');
+    queries.set(parsedQueries);
+    const search = getSearch(url);
```

Comment:
These params would be populated if a user is filtering on messages and then opens the create message wizard. I don't think these should be passed to the list providers call.